dolphin/drivers/manager.py

```diff
@@ -12,20 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import six
+import copy
 import stevedore
+import threading
 
 from oslo_log import log
-from oslo_utils import uuidutils
-
 from dolphin import exception
 from dolphin.i18n import _
 from dolphin import utils
+from dolphin.drivers import helper
```

Comment:
it's more clear to place import in alphabetical order, so put it front of `from dolphin import exception` :)

Reply:
Formatted.